CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v3.2.0 - 2017-07-03
+
+* KeyframeBot is designed to automate the process of validating keyframes and moving them to the chosen environment. [Heds Simons]
```

Comment:
It's the first instance of KeyframeBot and is adding it. The prefix is part of the sentence in this case. I could change this to a prefix I guess. I've been using `bot:` because usually it's easier than creating a sentence that has to parse properly with the botname as the first word...